### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by rglanz-rc**
Main README and MagicWeather README updated to be consistent with docs in purchases-android repository.

### Checklist
- [ x ] If applicable, unit tests
- [ x ] If applicable, create follow-up issues for `purchases-android` and hybrids

### Motivation

These are minor formatting tweaks that will make the READMEs more consistent with the purchases-android READMEs.

### Description

Most changes include moving the order of the information or adding a link to docs.revenuecat.com

I don't know how to test a readme 🙃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h3 align="center">😻 In-App Subscriptions Made Easy 😻</h1>
+<h3 align="center">😻 In-App Subscriptions Made Easy 😻</h3>
```
Comment (NachoSoto) [inline README.md:1]: Oops 😳

File: README.md [hunk 2]
```diff
@@ -39,14 +39,21 @@ Sign up to [get started for free](https://app.revenuecat.com/signup).
 🔀 | [Integrations](https://www.revenuecat.com/integrations) - over a dozen integrations to easily send purchase data where you need it
 💯 | Well maintained - [frequent releases](https://github.com/RevenueCat/purchases-ios/releases)
 📮 | Great support - [Help Center](https://community.revenuecat.com)
+🤩 | Awesome [new features](https://trello.com/b/RZRnWRbI/revenuecat-product-roadmap)
```
Comment (NachoSoto) [inline README.md:42]: Is this public? At the very least looks like it makes you log-in to see it.

Comment (NachoSoto) [reply README.md:42]: Oh yeah:
Screenshot 2022-08-19 at 13 35 15

File: README.md [hunk 3]
```diff
@@ -39,14 +39,21 @@ Sign up to [get started for free](https://app.revenuecat.com/signup).
...
 
-Or browse our iOS sample apps:
+Please follow the [Quickstart Guide](https://docs.revenuecat.com/docs/) for more information on how to install the SDK.
+
+Or view our iOS sample apps:
 - [MagicWeather](Examples/MagicWeather)
 - [MagicWeather SwiftUI](Examples/MagicWeatherSwiftUI)
 
+## Requirements
+- XCode 10.2+
```
Comment (NachoSoto) [inline README.md:54]:
```suggestion
- Xcode 10.2+
```

🙊

Comment (NachoSoto) [reply README.md:54]: But we require Xcode 13.2 now.

**Review by NachoSoto — COMMENTED** [review]
Nice!

File: README.md (hunk 1 above)
Comment (rglanz-rc) [reply README.md:1]: Still worked...

File: README.md (hunk 2 above)
Comment (rglanz-rc) [reply README.md:42]: I can take this off here and purchases-android. Unless it's meant to be enticing...?

File: README.md (hunk 1 above)
Comment (NachoSoto) [reply README.md:1]: I wonder if we should use `###` instead?

File: README.md (hunk 3 above)
Comment (rglanz-rc) [reply README.md:54]: 👍

Will that affect the minimum target (below)?

File: README.md (hunk 1 above)
Comment (rglanz-rc) [reply README.md:1]: I don't think (?) you can center without html on github

File: README.md (hunk 2 above)
Comment (aboedo) [reply README.md:42]: LOL that's... ehem... _dated_. 
As in, it's been closed for like 2 years. We should remove it

File: README.md (hunk 3 above)
Comment (aboedo) [reply README.md:54]: Not directly, but that deployment target is outdated too. 

You can see our current deployment target specs here: 
https://github.com/RevenueCat/purchases-ios/blob/main/RevenueCat.podspec#L17

That means Xcode 13 + iOS 11.0+, tvOS 11.0+, macOS 10.13+, watchOS 6.2+

File: README.md (hunk 2 above)
Comment (rglanz-rc) [reply README.md]: Removed.

File: README.md (hunk 3 above)
Comment (rglanz-rc) [reply README.md]: Changed to Xcode to 13.2+ and min iOS to 11.0+

Comment (aboedo) [reply README.md]: Could we mention the whole group? i.e.: iOS, tvOS, macOS, watchOS with the targets I mentioned before

Comment (rglanz-rc) [reply README.md]: Made a table for those platforms under Requirements

**Review by NachoSoto — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (NachoSoto) [reply README.md:1]: Oooh I didn't realize this is centered too, never mind.

**Pull request merged**